appserver/payara-appserver-modules/hazelcast-ejb-timer/src/test/java/fish/payara/ejb/timer/hazelcast/HazelcastTimerStoreTestBase.java

```diff
@@ -0,0 +1,10 @@
+package fish.payara.ejb.timer.hazelcast;
+
+import java.util.Collection;
+
+public abstract class HazelcastTimerStoreTestBase {
+  public String[] callListTimers(Collection<HZTimer> timers, String[] serverIds) {
```

Comment:
`serverIds` can be made a varargs which helps readability in calling methods avoiding the explicit array creation.